src/EditorFeatures/Test2/IntelliSense/QuickInfoControllerTests.vb

```diff
@@ -22,24 +24,25 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             TestWorkspace.ResetThreadAffinity()
         End Sub
 
-        <WpfFact>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
         Public Sub InvokeQuickInfoWithoutDocumentShouldNotQueryProviders()
             Dim emptyProvider = New Mock(Of IDocumentProvider)
             emptyProvider.Setup(Function(p) p.GetDocumentAsync(It.IsAny(Of ITextSnapshot), It.IsAny(Of CancellationToken))).Returns(Task.FromResult(DirectCast(Nothing, Document)))
             Dim controller As Controller = CreateController(documentProvider:=emptyProvider, triggerQuickInfo:=True)
             controller.WaitForController()
 
-            GetMocks(controller).Provider.Verify(Function(p) p.GetItemAsync(It.IsAny(Of Document), It.IsAny(Of Integer), It.IsAny(Of CancellationToken)), Times.Never)
+            'GetMocks(controller).Provider.Verify(Function(p) p.GetItemAsync(It.IsAny(Of Document), It.IsAny(Of Integer), It.IsAny(Of CancellationToken)), Times.Never)
+            Assert.Equal(0, If(TryCast(GetMocks(controller).Service, ServiceMock)?.InvokeCount, 0))
```

Comment:
TryCast seems odd: if that fails, the test magically passes.

Reply:
The TryCast here seems weird. This also came from the PR I inherited, but I didn't notice it. I'll tweak this to be a bit more sensible.

Reply:
fixed